pkg/module/security_scan.go

```diff
@@ -117,11 +126,12 @@ type SecurityScanCfg struct {
 	ModuleName  string            `json:"module-name"`
 	Protecode   []string          `json:"protecode"`
 	WhiteSource WhiteSourceSecCfg `json:"whitesource"`
+	DevBranch   string            `json:"dev-branch"`
```

Comment:
now we can test the security scan label in [e2e](https://github.com/kyma-project/cli/blob/main/tests/e2e/kyma_create_module_test.go)

This test use template-operator [security scan config file](https://github.com/kyma-project/template-operator/blob/main/sec-scanners-config.yaml), please update it with this new format first.